Buildings/Applications/DHC/package.mo

```diff
@@ -7,4 +7,4 @@ annotation (preferredView="info", Documentation(info="<html>
 This package contains models for district energy system.
 </p>
 </html>"));
-end DHC;
+end DHC;
```

Comment:
at the end of the line, needs to change to new line. It means there will be a empty line below it.

Reply:
Done